Answers.md

```diff
@@ -1 +1,18 @@
+Describe the biggest difference between .forEach & .map.
+forEach() iterates over an array and executes a provided function once each.
+While map() creates a new array with the results
 
+What is the difference between a function and a method?
+Method is a function when object is associated with it
+
+What is closure?
+Used to make global variables local
+
+Describe the four rules of the 'this' keyword.
```

Comment:
Window - default binding of the 'this' keyword
Implicit - method invocation binding of 'this' happens automatically with objects and methods
Explicit - uses function methods: call, apply, bind to explicitly bind 'this' to a function
New - used to create new objects and pass 'this' into constructor functions